pkg/spec/resources.go

```diff
@@ -38,6 +38,8 @@ import (
 	_ "k8s.io/client-go/pkg/apis/extensions/install"
 )
 
+const appLabelKey = "app"
```

Comment:
It might be better to create new package called `constaints` and put this there.
We can concentrate all constants in that package.
Also, it will be more readable if it is used as `constants.AppLabelKey`

+ this definitely needs a comment. It is not that clear on the first read what it is happening there with `appKeyValueToMap` and this constant.

Reply:
added comment, let's create the constants package in another PR